### PR TITLE
Update schema.graphql

### PR DESCRIPTION
**Pull request by PaulieB14**
Added descriptions to Schema

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated the docs to reflect my changes if applicable
- [x] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

<!--

### CODE REVIEW

**Comment by julien51** [comment]
Ho @PaulieB14 This is _very_ cool! Please open another PR and add yourself to the CLA file so we can approve that one!

**Comment by julien51** [comment]
It needs to be in _another_ PR :/

File: CLA.txt [hunk 1]
```diff
@@ -109,3 +109,5 @@ and interest in and to Your Contributions.
 8. You agree to notify Unlock of any facts or circumstances of
    which you become aware that would make these representations
    inaccurate in any respect.
+
+PauliB14
```
Comment (julien51) [inline CLA.txt:113]: ha no that's in this file: https://github.com/unlock-protocol/unlock/blob/master/.clabot

File: subgraph/schema.graphql [hunk 2]
```diff
@@ -1,43 +1,75 @@
 type Lock @entity {
+  "Transaction hash"
```
Comment (julien51) [inline subgraph/schema.graphql:2]:
```suggestion
  "Unique ID for the Lock object (uses the lock address)"
```

File: subgraph/schema.graphql [hunk 3]
```diff
@@ -1,43 +1,75 @@
...
   id: ID!
+  "Address of the lock"
   address: Bytes!
+  "A descriptive name for a collection of NFTs in this contract"
   name: String
+  "Gets the token symbol"
```
Comment (julien51) [inline subgraph/schema.graphql:8]:
```suggestion
  "Token symbol" 
```

File: subgraph/schema.graphql [hunk 4]
```diff
@@ -1,43 +1,75 @@
...
   address: Bytes!
+  "A descriptive name for a collection of NFTs in this contract"
   name: String
+  "Gets the token symbol" 
   symbol: String
+  "Duration is set the on the lock when you deploy and the expiration which is set on each key when they are minted"
   expirationDuration: BigInt
+  "Unique adddress for each token"
   tokenAddress: Bytes!
+  "Price of the lock"
```
Comment (julien51) [inline subgraph/schema.graphql:14]:
```suggestion
  "Price of the keys sold by the lock" 
```

File: subgraph/schema.graphql [hunk 5]
```diff
@@ -1,43 +1,75 @@
...
   expirationDuration: BigInt
+  "Unique adddress for each token"
   tokenAddress: Bytes!
+  "Price of the lock" 
   price: BigInt!
+  "An assigned role set on a Lock contract which gives the highest level of permissions to the wallet address set to that role"
   lockManagers: [Bytes!]!
+  "Unlock Protocol version of a minting contract"  
   version: BigInt!
+  "Number of keys owned by keyOwner (expired or not)"
```
Comment (julien51) [inline subgraph/schema.graphql:20]:
```suggestion
  "Number of keys minted (expired or not)"
```

File: subgraph/schema.graphql [hunk 6]
```diff
@@ -1,43 +1,75 @@
...
   totalKeys: BigInt!
+  "Setting for the max number of keys"
```
Comment (julien51) [inline subgraph/schema.graphql:22]:
```suggestion
  "Maximum number of keys for sale" 
```

File: subgraph/schema.graphql [hunk 7]
```diff
@@ -1,43 +1,75 @@
...
   totalKeys: BigInt!
+  "Setting for the max number of keys" 
   maxNumberOfKeys: BigInt
+ "The maximum number of key allowed for a single address"
   maxKeysPerAddress: BigInt
+  "Refer to key entity"
   keys: [Key!] @derivedFrom(field: "lock")
+  "Which block the lock was created"
   createdAtBlock: BigInt
+  "The block the last key was minted"
```
Comment (julien51) [inline subgraph/schema.graphql:30]:
```suggestion
  "The timestamp of the block in which the last key was minted"
```

File: subgraph/schema.graphql [hunk 8]
```diff
@@ -1,43 +1,75 @@
...
   lastKeyMintedAt: BigInt
 }
 
 type Key @entity {
+  "Transaction hash"
```
Comment (julien51) [inline subgraph/schema.graphql:35]:
```suggestion
  "Unique identifier for a key (combination of lock address and token id)" 
```

File: subgraph/schema.graphql [hunk 9]
```diff
@@ -1,43 +1,75 @@
...
+  "The block the last key was minted"
   lastKeyMintedAt: BigInt
 }
 
 type Key @entity {
+  "Transaction hash" 
   id: ID!
+  "In the Unlock ecosystem, a “Lock” is a smart contract that creates (or “mints”) NFTs"
   lock: Lock!
+  "TokenId for a given user"
```
Comment (julien51) [inline subgraph/schema.graphql:39]:
```suggestion
  "TokenId for a given key" 
```

File: subgraph/schema.graphql [hunk 10]
```diff
@@ -1,43 +1,75 @@
...
 type Key @entity {
+  "Transaction hash" 
   id: ID!
+  "In the Unlock ecosystem, a “Lock” is a smart contract that creates (or “mints”) NFTs"
   lock: Lock!
+  "TokenId for a given user" 
   tokenId: BigInt!
+ "The address of the key owner"
   owner: Bytes!
+  "An assigned title set on an Unlock key which gives a specific wallet address authorization to transfer, share or cancel"
```
Comment (julien51) [inline subgraph/schema.graphql:43]:
```suggestion
  "Address of the key owner"
```

File: subgraph/schema.graphql [hunk 11]
```diff
@@ -1,43 +1,75 @@
...
+  "The tokenURI on an NFT is a unique identifier"
   tokenURI: String
+  "Block key was created"
   createdAtBlock: BigInt!
+  "Invoked by a Lock manager to expire the user's key and perform a refund and cancellation of the keyThrows"
   cancelled: Boolean
 }
 
 type UnlockDailyData @entity {
+  "Transaction hash"
```
Comment (julien51) [inline subgraph/schema.graphql:56]:
```suggestion
  "Day identifier" 
```

File: subgraph/schema.graphql [hunk 12]
```diff
@@ -1,43 +1,75 @@
...
+  "Block key was created"
   createdAtBlock: BigInt!
+  "Invoked by a Lock manager to expire the user's key and perform a refund and cancellation of the keyThrows"
   cancelled: Boolean
 }
 
 type UnlockDailyData @entity {
+  "Transaction hash" 
   id: ID!
+  "Create lock This deploys a lock for a creator. It also keeps track of the deployed lock"
```
Comment (julien51) [inline subgraph/schema.graphql:58]:
```suggestion
  "Number of locks deployed on that dat"
```

File: subgraph/schema.graphql [hunk 13]
```diff
@@ -1,43 +1,75 @@
...
   owner: Bytes!
+  "An assigned title set on an Unlock key which gives a specific wallet address authorization to transfer, share or cancel"
   manager: Bytes
+  "Time the key expires"  
   expiration: BigInt!
+  "The tokenURI on an NFT is a unique identifier"
   tokenURI: String
+  "Block key was created"
   createdAtBlock: BigInt!
+  "Invoked by a Lock manager to expire the user's key and perform a refund and cancellation of the keyThrows"
```
Comment (wonderwomancode) [inline subgraph/schema.graphql:51]: "keyThrows" should be "key"

File: subgraph/schema.graphql [hunk 14]
```diff
@@ -1,43 +1,75 @@
...
 type UnlockDailyData @entity {
+  "Day identifier" 
   id: ID!
+  "Number of locks deployed on that day"
   lockDeployed: BigInt!
+  "Daily number of keys sold" 
   keysSold: BigInt!
+  "Daily number of active locks" 
   activeLocks: [Bytes!]
+  "Total value exchanged in the network"
```
Comment (wonderwomancode) [inline subgraph/schema.graphql:64]: "in" should be "on"

**Review by wonderwomancode — CHANGES_REQUESTED** [review]
Few minor edits :-)

File: subgraph/schema.graphql [hunk 15]
```diff
@@ -1,43 +1,75 @@
...
   price: BigInt!
+  "An assigned role set on a Lock contract which gives the highest level of permissions to the wallet address set to that role"
   lockManagers: [Bytes!]!
+  "Unlock Protocol version of a minting contract"  
   version: BigInt!
+  "Number of keys minted (expired or not)"
   totalKeys: BigInt!
+  "Maximum number of keys for sale" 
   maxNumberOfKeys: BigInt
+ "The maximum number of key allowed for a single address"
```
Comment (julien51) [inline subgraph/schema.graphql:24]:
```suggestion
 "The maximum number of keys allowed for a single address"
```

**Pull request merged**